### DISCUSSION TITLE
How NOT to override server rendered content?

### DISCUSSION THREAD

**Discussion by mrceperka** [sveltejs/kit#10676] [Q&A]
I have a page with a chart. The page is server rendered. I would like to disable rendering the chart again on the client. Unless user wants to (for example clicks the button). I was not able to achieve this, because the client page always overrides what was server from server.

I can't disable CSR on this page.

How do I do it in svelte kit?

**Comment by Fasteroid** [comment]
I didn't find an answer for this either—I had to write some ugly prebuild scripts to get the client and server synchronized.

**Comment by teemingc** [comment]
You'll have to provide some sample code for us to figure out the issue.

**Reply by mrceperka** [reply]
I'd say no need. Just imagine, that you render for example chart on the sever (as SVG) and then you don't want to rerender it on the client again until user clicks that chart.

Generally this question is about if it is possible to skip CSR for some part of the page.

I don't have a code sample right now, sorry and thank you for your interest :pray:

**Reply by teemingc** [reply]
When SSR occurs, the page is sent as HTML and CSR does not occur again on the client except for browser only code e.g, onMount, so it’s hard to tell what’s happening in your case without a reproduction
